cmd/cmdtest/testutils.go

```diff
@@ -31,6 +31,8 @@ import (
 
 const CLEANUP = true
 
+const TestDirPath = "../cmd/testdata"
```

Comment:
Best to use `filepath.Join()` here too. You probably can't use `const` then, but that's okay. Rather have it work on multiple platforms.

Reply:
Also, I think all references to "testdata" below can also use this `TestDirPath`

Reply:
Oops, missed this! Yeah I initially did this for `const`, but of course it makes more sense to make it platform unspecific.